keyboards/fortitude60/rules.mk

```diff
@@ -0,0 +1,72 @@
+SRC += matrix.c \
+       split_util.c \
+       serial.c
+
```

Comment:
Why no i2c support?

Reply:
There are several reasons, the biggest reason is that there is no more I / O port on this MCU board. It was filled with serial communication and LED backlight.

https://ja.aliexpress.com/item/Beetle-Virtual-Keyboard-BadUSB-Pro-Micro-ATMEGA32U4-Module-Mini-Development-Expansion-Board-For-Arduino-Leonardo-R3/32847715254.html?spm=a2g0s.9042311.0.0.fN4Uw8

This MCU board is about $5 with Aliexpress, but not as good as ProMicro, but you can easily get it. It is necessary to use USB Type-C which is the feature of this keyboard.
https://twitter.com/Pekaso/status/991295008912195584/

Reply:
Well, it's not the only solution for this.   In fact, something like this would have worked: https://www.adafruit.com/product/2910

But yeah, I can see that. Limited number of pins to physically use. 

(I'm only trying to be helpful here, but next time, feel free to jump on our discord and ask. There is a hardware section and number of very helpful people there that could assist with design elements like this).

But otherwise, looks good. :)